content/02.main-text.md

```diff
@@ -2,7 +2,7 @@
 
 The internet enables science to be shared in real-time at a low cost to a global audience.
 This development has decreased the barriers to making science open, while supporting new massively collaborative models of research.
-However, the scientific community requires tools whose workflows encourage openness.
+However, the scientific community requires tools whose workflows encourage openness [@doi:10.17226/25116].
 Manuscripts are the cornerstone of scholarly communication, but drafting and publishing manuscripts has traditionally relied on proprietary or offline tools that do not support _open scholarly writing_, in which anyone is able to contribute and the contribution history is preserved and public.
```

Comment:
Do we need support for the claim `drafting and publishing manuscripts has traditionally relied on proprietary or offline tools that do not support _open scholarly writing_`?  It seems self-evident, but https://doi.org/10.1038/514127a (which we already cite) could be relevant.
> Writing a paper is traditionally a stepwise process. One author shares drafts with colleagues and then waits for everyone to reply or moves forward independently, folding in revisions and queries as they arrive.

> A minority of researchers use these apps [_shareLaTeX, Fidus Writer and Authorea_]